src/renderers/dom/server/ReactServerRenderingTransaction.js

```diff
@@ -7,12 +7,14 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactServerRenderingTransaction
+ * @flow
```

Comment:
Similarly, let’s remove `@flow` here since we’re not adding complete coverage to this file.